bin/bugsnag-maze-runner

```diff
@@ -75,12 +122,51 @@ class MazeRunnerEntry
     @args = args.dup
     options = @parser.parse args
 
+    # Process CL options
     init if options[:init]
-    MazeRunner.configuration.appium_session_isolation = true if options[OPTION_SEPARATE_SESSIONS]
+    MazeRunner.configuration.appium_session_isolation = options[OPTION_SEPARATE_SESSIONS]
+    MazeRunner.configuration.app_location = options[OPTION_APP]
+    farm = options[OPTION_FARM]
+    device_type = MazeRunner.configuration.device_type = options[OPTION_DEVICE]
+    MazeRunner.configuration.farm = if farm.nil?
+                                      :none
```

Comment:
Should the default value of `farm` be set to `:none` in `Configuration.init`?

Reply:
It saves a couple of lines, but a `!farm.nil?` has to guard the final `raise`, the reader loses the immediate knowledge that `Configuration.farm` ends up as `:none` and an initializer has to be added to`Configuration`.  
If there was already default configuration in play then I'd probably change it, but on balance I think this is clearer.  Just.